app/core/less/variables.less

```diff
@@ -22,12 +22,15 @@
 @gray:                   #cccccc; // Container on gray background
 @gray-light:             #e6e6e5; // Container on white background
 @gray-lighter:           #f7f7f8; // Background
+@white-color: 			 #ffffff;
+@black-color: 			 #000;
```

Comment:
Minor thing, but spacing is wonky on the above two lines. Looks like tabs instead of spaces.